docs/getting-started/introduction.md

```diff
@@ -82,3 +82,147 @@ with Workflow(generate_name="dag-diamond-", entrypoint="diamond") as w:
         # We can use the `>>` or `.next()` operators to define dependencies between tasks.
         A >> [B, C] >> D
 ```
+
+## Comparison
+
+There have been other libraries available for structuring and submitting Argo Workflows:
+
+- [Couler](https://github.com/couler-proj/couler), which aimed to provide a unified interface for constructing and
+  managing workflows on different workflow engines. It has now been unmaintained since its last commit in April 2022.
+- [Argo Python DSL](https://github.com/argoproj-labs/argo-python-dsl), which allows you to programmatically define Argo
+  worfklows using Python. It was archived in October 2021.
+
+While the aforementioned libraries provided amazing functionality for Argo workflow construction and submission, they
```

Comment:
@flaviuvadan / @elliotgunton should we write a line or two about Bloomberg's involvement with hera and the conceptualize of Hera v5 and Hera + pydantic design? I would like hera to lean into the pydantic model more and more in the future the same way fastapi does to provide python native and intuitive interfaces.

Reply:
Sounds good - we could add some context about Bloomberg's involvement around the "The main reason for this is that v5 is a complete rewrite of the library" line. There's supposed to be a [pydantic support section](https://hera.readthedocs.io/en/stable/getting-started/walk-through/pydantic-support/) in the walkthrough (that I didn't get round to writing yet) so we can expand on Pydantic stuff there? Maybe move it into user guides as well

Reply:
@samj1912 have you seen tags like "organizations with major contributions to the project" in any OS project's README's? I wonder if we can have something like that